topics/single-cell/tutorials/scrna-data-ingest/tutorial.md

```diff
@@ -0,0 +1,881 @@
+
+Our Seurat starting file was generated by pulling data from [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc/home) and transforming it into the desired format, what was shown in the [data import tutorial]({% link topics/single-cell/tutorials/EBI-retrieval/tutorial.md %}). It will be our toy dataset.
+
+> <hands-on-title>Get toy data</hands-on-title>
+>
+> 1. Create a new history for this tutorial 
+> 2. Import the AnnData object from [Zenodo](https://zenodo.org/records/10397653)
+>
+>    ```
+>    https://zenodo.org/records/10397653/files/Seurat_object.rdata
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    
+> 3. Alternatively, you can import history where we created the Seurat object: [Input history](https://singlecell.usegalaxy.eu/u/j.jakiela/h/ebi-scxa---anndata-scanpy-or-seurat-object-1) 
+>
+>    {% snippet faqs/galaxy/histories_import.md %}
+>
+>
+> 4. **Rename** {% icon galaxy-pencil %} the dataset `Seurat object`
+>
+{: .hands_on}
+
+
+> <hands-on-title> Seurat to AnnData with SCEasy </hands-on-title>
+>
+> 1. {% tool [SCEasy convert](toolshed.g2.bx.psu.edu/repos/ebi-gxa/sceasy_convert/sceasy_convert/0.0.5+galaxy1) %} with the following parameters:
+>    - *"Direction of conversion"*: `Seurat to AnnData`
+>    - {% icon param-file %} *"Input object in Seurat RDS format"*: `Seurat object`
+>      (if the dataset does not show up in the corresponding input field, just drag the dataset from the history panel and drop into the input field)
+>    - *"Name of the assay to be transferred"*: `RNA`
+>    - *"Data type of the assay to be transferred"*: `data`
+>   
+> 2. **Rename** {% icon galaxy-pencil %} the output `Converted EBI-compatible AnnData file`.
+>
+> 3. Check that the datatype is `h5ad`
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="h5ad" %}
+>    
+{: .hands_on}
+
+And that's it! The latest tool {% tool [SCEasy Converter](toolshed.g2.bx.psu.edu/repos/iuc/sceasy_convert/sceasy_convert/0.0.7+galaxy2) %} will do the same but the output file will be the newest AnnData version and will not work with the tool used below.  
+
+Now, if we do small modification to metadata, we can use the generated dataset as an input file in the workflow we created in the [previous tutorial]({% link topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.md %})! Let's see how it works:
+
+> <hands-on-title> Modify AnnData object to make it compatible with Filter, Plot, Explore workflow </hands-on-title>
+>
+> 1. {% tool [AnnData Operations](toolshed.g2.bx.psu.edu/repos/ebi-gxa/anndata_ops/anndata_ops/1.8.1+galaxy92) %}
```

Comment:
Did it work for you? It failed for me.

Reply:
Oh no! 
It did work for me: https://singlecell.usegalaxy.eu/u/j.jakiela/h/seurat---anndata - see dataset 266 old SCEasy conversion, then 268 Anndata operations and all datasets that come later are the outputs of the Filter, Plot, Explore workflow.